tests/integration/test_spans.py

```diff
@@ -1230,8 +1225,8 @@ def test_span_ingestion_with_performance_scores(
                         },
                         "profile_id": "3d9428087fda4ba0936788b70a7587d0",
                         "op": "ui.interaction.click",
-                        "span_id": "bd429c44b67a3eb1",
-                        "segment_id": "968cff94913ebb07",
+                        "span_id": "cd429c44b67a3eb1",
+                        "segment_id": "cd429c44b67a3eb1",
```

Comment:
I checked some real `ui.interaction.click` spans in the product and they all have `span_id == segment_id`.